user_docs/en/userGuide.t2t

```diff
@@ -1620,6 +1620,34 @@ In this case, braille will not follow the NVDA navigator during object navigatio
 If you want braille to follow object navigation and text review instead, you need to configure braille to be tethered to review.
 In this case, Braille  will not follow system focus and system caret.
 
+==== Move system caret when Routing review cursor ====[BrailleSettingsReviewRoutingMovesSystemCaret]
+: Default
+  Never
+: Options
+  Default (Never), Never, Only when tethered automatically, Always
+:
+
+This setting determines if the system caret should be also tried to be moved
+with a routing button press when [braille tethering #BrailleTether] is set to automatic
+and braille is temporarily tethered to review, or tethering is explicitly set to the review cursor.

```

Comment:
I think this is easier to read. Also sentences shouldn't be split into new lines, it is apparently bad for translators.

```suggestion
This setting determines if the system caret should also be moved with a routing button press.
It occurs when [braille tethering #BrailleTether] is set to automatic and braille is temporarily tethered to review, or tethering is explicitly set to the review cursor.
```